source/draw/Image.ooc

```diff
@@ -27,6 +27,11 @@ CoordinateSystem: enum {
 	YUpward = 0x02
 }
 
+TransformMethod: enum {
```

Comment:
@emilwestergren , do you think there's a way we could use this in the GpuImages? I'm not asking for it to happen right now, obviously.